src/components/accordion-assignment-sheet/AccordionAssignmentSheet.tsx

```diff
@@ -0,0 +1,211 @@
+  const onSubmit = async () => {
+    setIsLoading(true);
+
+    // Submit the form data
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+    setTimeout(() => {
+      setIsLoading(false);
+      toast({
+        variant: 'default',
+        description: 'Updated successfully!',
+      });
+    }, 1000);
+  };
+
+  return (
+    <>
+      {isLoading && <LoadingSpinner isLoading={true} />}
+      <Accordion
+        type="single"
+        collapsible
+        className="w-full"
+        defaultValue={`item-1`}
+      >
+        <form onSubmit={handleSubmit(onSubmit)}>
+          {Object.entries(rotationData.rotations).map(([key, rotation]) => {
+            const numericKey = parseInt(key);
+            if (numericKey > 2) {
+              return null;
+            }
+            return (
+              <AccordionItem value={`item-${key}`} key={key}>
+                <AccordionTrigger>
+                  {rotation.startDate} - {rotation.endDate}
+                </AccordionTrigger>
+                <AccordionContent
+                  className={'bg-primary-lightest p-0 rounded-none'}
+                  asChild
+                >
+                  {rotation.categories.map((category) => (
+                    <div key={category.id}>
+                      <label className="flex items-center gap-1 mb-2 font-medium cursor-pointer">
+                        <Checkbox
+                          checked={category.tasks.every(
+                            (task) => isChecked[task.id],
+                          )}
+                          onCheckedChange={() =>
+                            handleAllCheckedChange(category.id)
+                          }
+                          disabled={numericKey > 1}
+                        />
+                        <p>{category.name}</p>
+                      </label>
+
+                      {category.tasks.map((task) => (
+                        <div
+                          key={task.id}
+                          className="flex justify-between items-center bg-white rounded-xl mb-2"
+                        >
+                          <label
+                            className={cn(
+                              `flex flex-1 items-center gap-1 py-2 px-3 cursor-pointer ${isChecked[task.id] ? 'text-gray-500' : ''}`,
+                            )}
+                          >
+                            <Checkbox
+                              checked={isChecked[task.id] || false}
+                              onCheckedChange={() =>
+                                handleCheckboxChange(task.id)
+                              }
+                              disabled={numericKey > 1}
+                            />
+                            <p>{task.title}</p>
+                          </label>
+                          <div className="flex items-center w-10 h-12 cursor-pointer">
+                            {numericKey < 2 && (
```

Comment:
IMO, using `numericKey === 1` is more clear and appropriate than `numericKey < 2`. I would appreciate it if you could make this change.

*Also, `LINE: 185`